ee/modules/160-multitenancy-manager/templates/user-resources/secure-project-template.yaml

```diff
@@ -0,0 +1,255 @@
+        {{ with .parameters.requests.storage }}requests.storage: {{ . }}{{ end }}
+        {{ with .parameters.limits.cpu }}limits.cpu: {{ . }}{{ end }}
+        {{ with .parameters.limits.memory }}limits.memory: {{ . }}{{ end }}
+    ---
+    # Deny all network traffic by default except namespaced traffic and dns.
+    # Refer to https://kubernetes.io/docs/concepts/services-networking/network-policies/
+    {{- if eq .parameters.networkPolicy "Isolated" }}
+    kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
+    metadata:
+      name: deny-all-except-current-namespace
+    spec:
+      podSelector:
+        matchLabels: {}
+      policyTypes:
+        - Ingress
+        - Egress
+      ingress:
+        - from:
+            - namespaceSelector:
+                matchLabels:
+                  kubernetes.io/metadata.name: "{{ .projectName }}"
+      egress:
+        - to:
+            - namespaceSelector:
+                matchLabels:
+                  kubernetes.io/metadata.name: "{{ .projectName }}"
+        - to:
+            - namespaceSelector:
+                matchLabels:
+                  kubernetes.io/metadata.name: kube-system
+          ports:
+            - protocol: UDP
+              port: 53
+    {{- end }}
+    ---
+    # Refer to https://deckhouse.io/documentation/v1/modules/460-log-shipper/
+    {{- with .parameters.clusterLogDestnationName }}
+    apiVersion: deckhouse.io/v1alpha1
+    kind: PodLoggingConfig
+    metadata:
+      name: whispers-logs
+      namespace: tests-whispers
+    spec:
+      clusterDestinationRefs:
+        - {{ . }}
+    {{- end }}
+    ---
+    # Loki ClusterLogDestination config example
+    # apiVersion: deckhouse.io/v1alpha1
+    # kind: ClusterLogDestination
+    # metadata:
+    #   name: loki-storage
+    # spec:
+    #   type: Loki
+    #   loki:
+    #     endpoint: http://loki.loki:3100
+    ---
+    # Enable audit rules (create a falco audit rule to record some malicious activities of the pods in a given namespace)
+    # Refer to https://deckhouse.io/documentation/v1/modules/650-runtime-audit-engine/examples.html
+    {{- if .parameters.enabledAudit }}
+    apiVersion: deckhouse.io/v1alpha1
+    kind: FalcoAuditRules
+    metadata:
+      name: ownership-permissions
+    spec:
+      rules:
+      - macro:
+          name: spawned_process
+          condition: (evt.type in (execve, execveat) and evt.dir=<)
+      - rule:
+          name: Detect Ownership Change
+          desc: detect file permission/ownership change
+          condition: >
+            spawned_process and proc.name in (chmod, chown) and proc.args contains "/tmp/"
+          output: >
+            The file or directory below has had its permissions or ownership changed (user=%user.name
+            command=%proc.cmdline file=%fd.name parent=%proc.pname pcmdline=%proc.pcmdline gparent=%proc.aname[2])
+          priority: Warning
+          tags: [filesystem]
```

Comment:
We need to add something more meaningful. Let's think about it.

Reply:
Replaced the template with an example of sending messages about shell is run in a container